### PR TITLE
avoid logging entire stack trace

### PR DESCRIPTION
**Pull request by shaharuk-yb**

### CODE REVIEW

File: src/main/java/com/oltpbenchmark/api/Worker.java [hunk 1]
```diff
@@ -456,13 +456,13 @@ protected final void doWork(DatabaseType databaseType, TransactionType transacti
                         conn.rollback();
 
                     if (isRetryable(ex)) {
-                        LOG.debug(String.format("Retryable SQLException occurred during [%s]... current retry attempt [%d], max retry attempts [%d], sql state [%s], error code [%d].", transactionType, retryCount, maxRetryCount, ex.getSQLState(), ex.getErrorCode()), ex);
+                        LOG.debug(String.format("Retryable SQLException occurred during [%s]... current retry attempt [%d], max retry attempts [%d], sql state [%s], error code [%d].", transactionType, retryCount, maxRetryCount, ex.getSQLState(), ex.getErrorCode()));
```
Comment (hbhanawat) [inline src/main/java/com/oltpbenchmark/api/Worker.java:459]: is this message getting dumped? This is a debug level message and logs should not be dumped by default. If it is not being dumped, please dont change it

File: src/main/java/com/oltpbenchmark/api/Worker.java [hunk 2]
```diff
@@ -456,13 +456,13 @@ protected final void doWork(DatabaseType databaseType, TransactionType transacti
...
 
                         status = TransactionStatus.RETRY;
 
                         retryCount++;
                     } else {
-                        LOG.warn(String.format("SQLException occurred during [%s] and will not be retried... sql state [%s], error code [%d].", transactionType, ex.getSQLState(), ex.getErrorCode()), ex);
+                        LOG.warn(String.format("SQLException occurred during [%s] and will not be retried... sql state [%s], error code [%d].", transactionType, ex.getSQLState(), ex.getErrorCode()));
```
Comment (hbhanawat) [inline src/main/java/com/oltpbenchmark/api/Worker.java:465]: Please add ex.getMessage() as well. It should be a short string.

**Review by hbhanawat — CHANGES_REQUESTED** [review]

File: src/main/java/com/oltpbenchmark/api/Worker.java (hunk 1 above)
Comment (shaharuk-yb) [reply src/main/java/com/oltpbenchmark/api/Worker.java]: this is not getting dumped. Removed this change.

File: src/main/java/com/oltpbenchmark/api/Worker.java (hunk 2 above)
Comment (shaharuk-yb) [reply src/main/java/com/oltpbenchmark/api/Worker.java]: added ex.getMessage()

**Review by hbhanawat — APPROVED** [review]

**Pull request merged**